CMakeLists.txt

```diff
@@ -1,3 +1,34 @@
-cmake_minimum_required (VERSION 2.6)
-project (Hello)
-add_executable(Hello src/main.cpp)
+cmake_minimum_required(VERSION 2.8.2)
+
+project(UnitTest)
```

Comment:
No reason to change the `project()` variable here, maybe you made some mistakes regarding the new CMakeLists